x/crconsent/client/cli/txRequest.go

```diff
@@ -0,0 +1,152 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/metabelarus/mbcorecr/x/crconsent/types"
+)
+
+func CmdCreateRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-request [initiator] [recipient] [requestType] [status] [value] [memo] [promoUrl] [creationDt] [finalDt]",
```

Comment:
Please extend use description with possible values of the fields. It's true for all cli commands. Especially for enum values.